app/javascript/packs/application.tsx

```diff
@@ -85,6 +86,9 @@ initReact({
     />
   ),
   'student-editor': (data: any) => <Student.Editor endpoint={data.endpoint} />,
+  'student-tooltip': (data: any) => (
+    <Tooltips.Student endpoint={data.endpoint} />
```

Comment:
I've tried to keep the arguments as simple as possible, just like the editor.

Reply:
Passing in urls === perfect.